templates/vereinsheim.html

```diff
@@ -42,7 +42,7 @@ <h2>{{ this.title }}</h2>
       $('.lightgallery').lightGallery();
     });
   </script>
-  <script src="{{ '/js/openstreetmap.js'|url }}"></script>
+  <script src="{{ '/js/openstreetmap.min.js'|url }}"></script>
```

Comment:
``|url`` kann man glaube ich durch ``|asseturl`` ersetzen.

ist eine ungetestete vermutung!

Gild auch für das folgende JavaScript!

Reply:
ja absolut das wollte ich eigentlich in einem weiteren Pull request überarbeiten da nicht wirklich zugehörig, kann ich aber auch schnell hier fixen

Reply:
so habe es für alle assets in den html files abgeändert